packages/suite-desktop/src-electron/libs/processes/BridgeProcess.ts

```diff
@@ -39,7 +40,7 @@ class BridgeProcess extends BaseProcess {
     }
 
     async startDev(): Promise<void> {
-        await this.start(['-e', '21324']);
+        await super.start(['-e', '21324']);
```

Comment:
super? shouldn't super be used by the same function name?

Reply:
I'm not sure actually. I thought `super` is the JavaScript way to refer to the parent class. Either way, both work but I'm not sure which way is more correct :smile:

Reply:
looks like its valid
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super#super-calling_static_methods

Reply:
I read up a little on it. `this` is used for referring the current instance of the class (which being extended from `BaseProcess`, has access to the `start` method). `super` is used for the referring to the parent class. The part where I'm not sure is about context and if the properties accessed with `this` or `super` might be different. Worth trying it out (or if someone can lecture me, please!) but for now I'll revert it back to this for safety.

Reply:
Addressed in  417a0f7.